src/cpp/session/modules/SessionUpdates.cpp

```diff
@@ -134,7 +134,14 @@ void endRPCUpdateCheck(const json::JsonRpcFunctionContinuation& cont,
 {
    json::JsonRpcResponse response;
    response.setResult(jsonFromProcessResult(result));
-   cont(Success(), &response);
+   if (result.stdErr.empty()) {
```

Comment:
Nicpick: Braces on separate line (our Java/C++ style guidelines: https://github.com/rstudio/rstudio/wiki/Coding-Standards-(java,-cpp))

Reply:
thanks Gary! I'll do a style guidelines pass for future contributions 👍